ROMFS/px4fmu_common/init.d/airframes/4041_beta75x

```diff
@@ -31,7 +31,7 @@ then
 
 	param set MC_AIRMODE 2
 	param set MC_PITCHRATE_D 0.0010
-	param set MC_PITCHRATE_I 0.4
+	param set MC_PITCHRATE_I 0.5
```

Comment:
Are `MC_ROLLRATE_I` kept intentionally? For these the same advantage is now applicable, right?

Reply:
Yes, but we can have another test with FPV, I only tested LOS. Generally, in my experience pitch I is always a bit higher than roll I.